spec/controllers/setup_controller_spec.rb

```diff
@@ -173,6 +173,106 @@
       end
     end
 
+    context "when suse registry mirror is disabled" do
+      let(:no_registry_mirror_settings) do
+        s = settings_params.dup
```

Comment:
MINOR: could write as well:

```ruby
settings_params.dup.tap do |s|
  ...
end
```

...but it's a matter of taste i guess

Reply:
I like the block syntax...